### DISCUSSION TITLE
MPJS import

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#97] [Polls]
Cruiser supports import of 4 route formats:
- GPX (offline navigation and normal)
- ITN
- Kurviger
- MPJS

MPJS format is from the Motoplaner website (closed on 2018): https://groups.google.com/g/kurviger/c/tistjvQ9R1A

Do you think we should continue having the import for .mpjs files or it is not needed anymore?

**Comment by MotoUKRider** [comment]
Speaking of file formats, the author of ITNConv has closed his website :(

**Comment by ramondevic** [comment]
I only use Kurviger and GPX.

**Reply by DolomitesTom** [reply]
Same here

**Comment by devemux86** [comment]
Thanks for the answers and votes, so we can remove it in the next Cruiser update.
